<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>411</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>127</width>
      <height>0</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="start_tab">
     <attribute name="title">
      <string>Start Page</string>
     </attribute>
     <attribute name="toolTip">
      <string>Welcome to CSV Editor's start page</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>236</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Google Sans</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CSV Editor's Start Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_page_btn_open_file">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Open File</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="btn_load_csv">
         <property name="text">
          <string>Load CSV</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>236</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Google Sans</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_page_btn_recent_files">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Recent Files</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Filename here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="main_document_tab">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <attribute name="title">
      <string>Main Document</string>
     </attribute>
     <widget class="QTableWidget" name="csv_data_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>611</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="plot_tab">
     <attribute name="title">
      <string>Plot</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>10</y>
        <width>541</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_save_plot">
         <property name="text">
          <string>Save Plot PNG</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close_plot">
         <property name="text">
          <string>Close Plot</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="input_plot_title">
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="placeholderText">
          <string>Plot Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_set_plot_title">
         <property name="text">
          <string>Set Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_plot_xy">
         <property name="text">
          <string>Plot (x,y)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_plot_yx">
         <property name="text">
          <string>Plot (y,x)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>571</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="plot_frame">
       <item>
        <layout class="QHBoxLayout" name="plot_frame_horizontal"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_load_file"/>
    <addaction name="action_close_file"/>
    <addaction name="separator"/>
    <addaction name="action_add_data"/>
    <addaction name="separator"/>
    <addaction name="action_save_file"/>
    <addaction name="action_save_plot_png"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_edit_data"/>
    <addaction name="action_delete_selected"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="action_plot_scatter_points"/>
    <addaction name="action_plot_scatter_points_lines"/>
    <addaction name="action_plot_lines"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_column_layout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_View"/>
   <addaction name="menuPlot"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_toolbar_open_file"/>
   <addaction name="action_toolbar_save_file"/>
   <addaction name="separator"/>
   <addaction name="action_toolbar_add_data"/>
   <addaction name="action_toolbar_delete_selected"/>
   <addaction name="separator"/>
   <addaction name="action_toolbar_plot_scatter_points"/>
   <addaction name="action_toolbar_plot_scatter_points_lines"/>
   <addaction name="action_toolbar_plot_lines"/>
   <addaction name="action_toolbar_save_plot_png"/>
  </widget>
  <widget class="QToolBar" name="toolbar_bottom_info">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
border: 2px solid;
border-color: transparent #ccc transparent transparent
}</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_toolbar_bottom_column_count"/>
   <addaction name="action_toolbar_bottom_row_count"/>
   <addaction name="action_toolbar_bottom_source"/>
   <addaction name="action_toolbar_bottom_column"/>
   <addaction name="action_toolbar_bottom_row"/>
   <addaction name="action_toolbar_bottom_selected_cells"/>
   <addaction name="action_toolbar_bottom_text_length"/>
  </widget>
  <action name="action_load_file">
   <property name="text">
    <string>&amp;Load File</string>
   </property>
  </action>
  <action name="action_close_file">
   <property name="text">
    <string>&amp;Close File</string>
   </property>
  </action>
  <action name="action_save_file">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_edit_data">
   <property name="text">
    <string>Edit Data</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_add_data">
   <property name="text">
    <string>&amp;Add Data Row</string>
   </property>
  </action>
  <action name="action_column_layout">
   <property name="text">
    <string>Show/Hide Columns</string>
   </property>
  </action>
  <action name="action_toolbar_add_data">
   <property name="icon">
    <iconset>
     <normaloff>icons/add_icon.png</normaloff>icons/add_icon.png</iconset>
   </property>
   <property name="text">
    <string>Add Data</string>
   </property>
   <property name="toolTip">
    <string>Add Row to the table</string>
   </property>
  </action>
  <action name="action_toolbar_delete_selected">
   <property name="icon">
    <iconset>
     <normaloff>icons/delete_icon.png</normaloff>icons/delete_icon.png</iconset>
   </property>
   <property name="text">
    <string>Delete Selected</string>
   </property>
   <property name="toolTip">
    <string>Delete Selected</string>
   </property>
  </action>
  <action name="action_toolbar_plot_scatter_points">
   <property name="icon">
    <iconset>
     <normaloff>icons/scatter_plot_icon.png</normaloff>icons/scatter_plot_icon.png</iconset>
   </property>
   <property name="text">
    <string>Plot Scatter Points</string>
   </property>
   <property name="toolTip">
    <string>Plot Scatter Points</string>
   </property>
  </action>
  <action name="action_toolbar_plot_scatter_points_lines">
   <property name="icon">
    <iconset>
     <normaloff>icons/scatter_line_plot_icon.png</normaloff>icons/scatter_line_plot_icon.png</iconset>
   </property>
   <property name="text">
    <string>Plot Scatter Points with Smooth Lines</string>
   </property>
   <property name="toolTip">
    <string>Plot Scatter Points with Smooth Lines</string>
   </property>
  </action>
  <action name="action_toolbar_plot_lines">
   <property name="icon">
    <iconset>
     <normaloff>icons/line_plot_icon.png</normaloff>icons/line_plot_icon.png</iconset>
   </property>
   <property name="text">
    <string>Plot Lines</string>
   </property>
   <property name="toolTip">
    <string>Plot Lines</string>
   </property>
  </action>
  <action name="action_toolbar_save_file">
   <property name="icon">
    <iconset>
     <normaloff>icons/save_icon.png</normaloff>icons/save_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
   <property name="toolTip">
    <string>Save current table to CSV file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_toolbar_open_file">
   <property name="icon">
    <iconset>
     <normaloff>icons/open_file_icon.png</normaloff>icons/open_file_icon.png</iconset>
   </property>
   <property name="text">
    <string>Open CSV File</string>
   </property>
   <property name="toolTip">
    <string>Open CSV File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_plot_scatter_points">
   <property name="text">
    <string>Scatter Points</string>
   </property>
  </action>
  <action name="action_plot_scatter_points_lines">
   <property name="text">
    <string>Scatter Points with Smooth Lines</string>
   </property>
  </action>
  <action name="action_plot_lines">
   <property name="text">
    <string>Lines</string>
   </property>
  </action>
  <action name="action_save_plot_png">
   <property name="text">
    <string>Save Plot as PNG</string>
   </property>
  </action>
  <action name="action_toolbar_save_plot_png">
   <property name="icon">
    <iconset>
     <normaloff>icons/save_plot_png_icon.png</normaloff>icons/save_plot_png_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save Plot PNG</string>
   </property>
   <property name="toolTip">
    <string>Save Plot as PNG image</string>
   </property>
  </action>
  <action name="action_delete_selected">
   <property name="text">
    <string>Delete Selection</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_column_count">
   <property name="text">
    <string>15</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_row_count">
   <property name="text">
    <string>32</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_source">
   <property name="text">
    <string>test_data.csv</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_column">
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_row">
   <property name="text">
    <string>5</string>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
  </action>
  <action name="action_toolbar_bottom_selected_cells">
   <property name="text">
    <string>15</string>
   </property>
  </action>
  <action name="action_toolbar_bottom_text_length">
   <property name="text">
    <string>18</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
